embed/etcd.go

```diff
@@ -691,12 +690,9 @@ func configureClientListeners(cfg *Config) (sctxs map[string]*serveCtx, err erro
 			sctx.userHandlers[k] = cfg.UserHandlers[k]
 		}
 		sctx.serviceRegister = cfg.ServiceRegister
-		if cfg.EnablePprof || cfg.Debug {
+		if cfg.EnablePprof {
```

Comment:
@gyuho Do we want to add `cfg.LogLevel == "debug"` to the if condition here? In 3.4, if user specifies `--log-level 'debug'`, this will not be triggered. On the other hand, do we want `--log-level 'debug'` to have the same behavior as the deprecated `--debug` flag?

Reply:
@jingyih Good catch.

Let's do `cfg.LogLevel == "debug"`.

> do we want --log-level 'debug' to have the same behavior as the deprecated --debug flag?

Yes, as we deprecate `--debug` flag, in favor of `log-level`.

This should be fine as long as we highlight in the CHANGELOG and upgrade guides.

Reply:
Done. Could you help take another look? Thanks!